src/Components/Admin/admin.js

```diff
@@ -133,13 +153,46 @@ export default function Admin() {
                 >
                     Sign Out
                 </Button>
+
                 <div className={classes.root}>
                     <IconButton
                         className={classes.addButton}
                         aria-label="delete"
+                        onClick={handleAddButtonClick}
                     >
                         <AddIcon />
                     </IconButton>
+                    <Menu
+                        anchorEl={anchorEl}
+                        anchorOrigin={{
+                            vertical: "top",
+                            horizontal: "center",
```

Comment:
This (and lines 172-173) can be extracted to a useStyles right?

Reply:
We could do that (not sure on the logistics) but we already had this pattern implemented as part of the navbar implementation. IMO if we are repeating this pattern throughout the component then yes, it would make sense to extract to useStyles but we are only using this behaviour once.